chains/evm/.gas-snapshot

```diff
@@ -505,4 +506,4 @@ USDCTokenPool_releaseOrMint:test_ReleaseOrMintRealTx() (gas: 265841)
 USDCTokenPool_supportsInterface:test_SupportsInterface() (gas: 10100)
 VersionedVerifierResolver_applyInboundImplementationUpdates:test_applyInboundImplementationUpdates() (gas: 109230)
 VersionedVerifierResolver_applyOutboundImplementationUpdates:test_applyOutboundImplementationUpdates() (gas: 108154)
-e2e:test_e2e() (gas: 465211)
\ No newline at end of file
+e2e:test_e2e() (gas: 450709)
```

Comment:
mostly due to unchecked msg decoding as everything else would make it more expensive